lib/transcryptor/instance.rb

```diff
@@ -33,10 +33,14 @@ def initialize(adapter)
   # * <tt>:algorithm</tt> - Encryption algorithm (default: +'aes-256-gcm'+).
   #
 
-  def re_encrypt(table_name, attribute_name, old_opts, new_opts, other_columns = [])
+  def re_encrypt(table_name, attribute_name, old_opts, new_opts, re_encrypt_opts = {})
     old_opts.reverse_merge!(transcryptor_default_options)
     new_opts.reverse_merge!(transcryptor_default_options)
 
+    other_columns = re_encrypt_opts.delete(:other_columns) || []
```

Comment:
`re_encrypt_opts.fetch(:other_columns, [])` and I do not see any reason to `#delete`.